### PR TITLE
new description for adding a WFS-Layer to the canvas

### PR DESCRIPTION
**Pull request by lajunek**
###New description for adding a WFS layer to the canvas

Load a WFS-Layer from DM Solutions

Ticket: #

- [ ] Backport to LTR documentation is required

### Minimal requirements for merging *(for maintainers)*

- [ ] The description of this PR is coherent with the manual and does not provide wrong information.
- [ ] This PR passes the checks.

### CODE REVIEW

**Comment by lajunek** [comment]
@hatav @DelazJ  what do you think about the new description?

File: source/docs/user_manual/working_with_ogc/ogc_client_support.rst [hunk 1]
```diff
@@ -617,14 +617,27 @@ The difference is that there are no default servers defined, so we have to add o
 **Loading a WFS Layer**
 
 As an example, we use the DM Solutions WFS server and display a layer.
-The URL is: http://www2.dmsolutions.ca/cgi-bin/mswfs_gmap
+https://demo.gatewaygeomatics.com/cgi-bin/wfs_gateway?REQUEST=GetCapabilities&VERSION=1.0.0&SERVICE=WFS
+
+To be able to load a WFS Layer we create a connection to the WFS server first.
 
 #. Click on the |wfs| :sup:`Add WFS Layer` tool on the
    Layers toolbar. The :guilabel:`Add WFS Layer from a Server` dialog appears.
```
Comment (DelazJ) [inline source/docs/user_manual/working_with_ogc/ogc_client_support.rst]: The "Layers" toolbar (actually "Manage Layers" toolbar) is hidden by default when installing QGIS and is meant (it was the plan at least) to be replaced with the Data Source Manager. Could you instead (or also) refer to the Datasource Manager and the Layer menu. See examples in https://docs.qgis.org/3.4/en/docs/user_manual/managing_data_source/opening_data.html or https://docs.qgis.org/3.4/en/docs/user_manual/managing_data_source/create_layers.html

File: source/docs/user_manual/working_with_ogc/ogc_client_support.rst [hunk 2]
```diff
@@ -617,14 +617,27 @@ The difference is that there are no default servers defined, so we have to add o
 **Loading a WFS Layer**
 
 As an example, we use the DM Solutions WFS server and display a layer.
```
Comment (havatv) [inline source/docs/user_manual/working_with_ogc/ogc_client_support.rst]: Is this still the DM Solutions' WFS server?

Comment (lajunek) [reply source/docs/user_manual/working_with_ogc/ogc_client_support.rst]: @hatav Actually not. I remove this information

Comment (havatv) [reply source/docs/user_manual/working_with_ogc/ogc_client_support.rst]: @lajunek - That would be fine.  Remember to also change the name of the layer in the rest of the document...

Comment (lajunek) [reply source/docs/user_manual/working_with_ogc/ogc_client_support.rst]: @DelazJ 
I commited the above suggestions in the same local branch I used for #3765. Will these changes be shown under #3765 on this page or will I need to open a new PR and merge this new PR. Also: do I have to merge the PR #3765 before creating the following PR (from the commit I made today)?

Comment (DelazJ) [reply source/docs/user_manual/working_with_ogc/ogc_client_support.rst]: > I commited the above suggestions in the same local branch I used for #3765. Will these changes be shown under #3765 on this page 

Doing the changes in the same `add_wfs_connection` branch is the right thing to do and once you pushed online, it'll be shown
> will I need to open a new PR and merge this new PR. Also: do I have to merge the PR #3765 before creating the following PR (from the commit I made today)?

No!  
For how things work, you may want to read https://docs.qgis.org/testing/en/docs/documentation_guidelines/first_contribution.html we've written for QGIS-Docs contributors

**Comment by DelazJ** [comment]
Euh... actually it was merged before I even replied.

**Pull request merged**